### PR TITLE
Mounted guns names, grenades tweaks and projectiles changes

### PR DESCRIPTION
**Pull request by Alberyk**
-name added the prefix mounted to all mounted guns
-converted the hos sniper rifle bullet to the new armor system
-converted projectile/special.dm to absolute pathing
-reduced again the damage of frag grenades
-added armor penetration to the pulse beam, to make it more than just a stronger laser
-fixed the fake grenade getting stuck in the primed sprite after its detonation and added a indication it is take in the desc

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -214,19 +218,19 @@
 			if(!isrobot(target))
 				L.apply_effect(5, DROWSY, 0)
 				if(def_zone == "torso")
-					if(blocked < 2 && !(blocked < 1))
+					if(blocked < 100 && !(blocked < 1))
 						target.visible_message("<b>[target]</b> yawns.")
-					if(blocked < 1)
+					if(blocked < 20)
 						spawn(120)
```
Comment (Lohikar) [inline code/modules/projectiles/projectile/bullets.dm]: Could you convert these three spawns to a timer please? Two are similar enough that they should be doable with a single proc that takes an argument.

File: code/modules/projectiles/projectile/special.dm [hunk 2]
```diff
@@ -94,36 +94,36 @@
...
-					else
-						randmutg(M)
-						domutcheck(M,null)
+/obj/item/projectile/energy/floramut/on_hit(var/atom/target, var/blocked = 0)
+	var/mob/living/M = target
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = M
+		if((H.species.flags & IS_PLANT) && (M.nutrition < 500))
+			if(prob(15))
+				H.apply_effect((rand(30,80)),IRRADIATE,blocked = H.getarmor(null, "rad"))
```
Comment (Lohikar) [inline code/modules/projectiles/projectile/special.dm]: Isn't this supposed to _deal_ damage to dionae? They heal from rads.

Comment (Alberyk) [inline code/modules/projectiles/projectile/special.dm]: No idea, I did not touch the original code.

File: code/modules/projectiles/projectile/bullets.dm [hunk 3]
```diff
@@ -243,6 +243,9 @@
 			M.Sleeping(1200)*/ //commented out until simple_mob paralysis actually works.
 	..()
 
+/obj/item/projectile/bullet/rifle/tranq/proc/apply_sedative(var/mob/living/L, var/severity)
```
Comment (Lohikar) [inline code/modules/projectiles/projectile/bullets.dm]: Maybe the emote should go in here for consistency's sake?

File: code/modules/projectiles/projectile/bullets.dm [hunk 4]
```diff
@@ -219,21 +219,21 @@
...
+						L.emote("yawns")
 					if(blocked < 20)
-						spawn(120)
-							L.apply_effect(10, PARALYZE, 0)
-							target.visible_message("<b>[target]</b> moans.")
+						L.emote("moans")
+						addtimer(CALLBACK(src, .proc/apply_sedative, target, 10), 120)
 				if(def_zone == "head" && blocked < 100)
+					addtimer(CALLBACK(src, .proc/apply_sedative, target, 20), 35)
 					spawn(35)
```
Comment (Lohikar) [inline code/modules/projectiles/projectile/bullets.dm]: Replace this one too if you can please.

**Pull request merged**